sql-plugin/src/main/311+-all/scala/com/nvidia/spark/rapids/shims/v2/ParquetCachedBatchSerializer.scala

```diff
@@ -553,6 +553,8 @@ class ParquetCachedBatchSerializer extends GpuCachedBatchSerializer with Arm {
     val rapidsConf = new RapidsConf(conf)
     val (cachedSchemaWithNames, selectedSchemaWithNames) =
       getSupportedSchemaFromUnsupported(cacheAttributes, newSelectedAttributes)
+    val origSelectedAttributesWithUnambiguousNames = 
```

Comment:
Just a nitpick. The result of `sanitizeColumnNames()` isn't used in all branches of the subsequent `if{}` statement. I wonder if we can't move this definition to the `else{}` block.